srcpkgs/llvm17/template

```diff
@@ -576,17 +585,26 @@ libomp-devel_package() {
 	short_desc+=" - Clang OpenMP support library - development files"
 	depends="libomp>=${version}_${revision} llvm17>=${version}_${revision}"
 	pkg_install() {
-		vmove usr/bin/llvm-omp-device-info
-		vmove usr/bin/llvm-omp-kernel-replay
-		vmove "usr/lib/libarcher*.so"
+		if [ -f "${DESTDIR}/usr/bin/llvm-omp-device-info" ]; then
+			vmove usr/bin/llvm-omp-device-info
+		fi
+		if [ -f "${DESTDIR}/usr/bin/llvm-omp-kernel-replay" ]; then
+			vmove usr/bin/llvm-omp-kernel-replay
+		fi
+		if [ -f "${DESTDIR}/usr/lib/libarcher.so" ]; then
+			vmove "usr/lib/libarcher*.so"
+		fi
+		if [ -f "${DESTDIR}/usr/lib/libarcher_static.a" ]; then
+			vmove "usr/lib/libarcher*.a"
```

Comment:
This file was already moved 3 lines earlier. You should match the `vmove` with your condition

Reply:
the shared `.so` was moved 3 lines earlier, this vmoves the static `.a`

Reply:
True.